.github/workflows/deploy-prod.yml

```diff
@@ -0,0 +1,33 @@
+# .github/workflows/deploy-prod.yml
+
+name: Deploy to Live Channel
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  preview:

```

Comment:
Maybe this job name should be "live" instead of "preview."

Reply:
Fixed!